tutorials/tutorial_kml.py

```diff
@@ -46,7 +46,7 @@ def automate_kml():
 
 
 def _switch_to_europe_map():
-    find_and_click_picture('topviewwindow-01-europe-cyl.png', "Map change dropdown could not be located on the screen.")
+    find_and_click_picture('topviewwindow-07-europe-cyl.png', "Map change dropdown could not be located on the screen.")
```

Comment:
this won't work

The image available is 
topviewwindow-00-global-cyl.png


it can only find the first on screen visible value on a pulldown.

![image](https://github.com/user-attachments/assets/c3e500a6-7711-48d1-abca-d355a3b62efe)

this can be verified by
```
$ export MSUI_CONFIG_PATH=/tmp/msui_tutorials
$ export XDG_CACHE_HOME=$(mktemp -d)
$ python
>>> from mslib.msui.msui import main
>>> main(tutorial_mode=True)
CTRL+H
CTRL+UP
CTRL+F
CTRL+Q
$ ls /tmp/msui-tutorials/tutorial_images/00* 
```
![topviewwindow-00-global-cyl](https://github.com/user-attachments/assets/03259fd7-4b87-44bd-8ab5-d95e3678a91f)